### PR TITLE
Finnish localization added

### PR DESCRIPTION
**Pull request by Newman101**
Finnish localization (YML file) added. Thanks. :)

### CODE REVIEW

File: locale/fi.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+en:
```
Comment (prat0318) [inline locale/fi.yml]: `en` will be replaced with `fi`

**Comment by prat0318** [comment]
Looks good. It will be great if README can be edited to add `fi` to the available locales. Thanks for contributing <3

**Comment by Newman101** [comment]
Sure, I will. You're welcome. :+1:

**Comment by Newman101** [comment]
README.md has been now edited and fi has been added to available locales. ^.^

**Pull request merged**